nutils/_graph.py

```diff
@@ -68,7 +69,33 @@ def export_graphviz(self, *, fill_color: Optional[GraphvizColorCallback] = None,
                 for i, line in enumerate(src.split('\n'), 1):
                     print('{:04d}  {}'.format(i, line))
                 treelog.warning('graphviz failed for error code', status.returncode)
-            img.write(status.stdout)
+            graph = status.stdout
+            if image_type == 'svg':
+                graph = re.sub(b'<svg (.*?)>', insert_clickHandler, graph, count=1, flags=re.DOTALL)
+            img.write(graph)
+
+insert_clickHandler = rb'''<svg \1 onload="document.addEventListener('click', clickHandler)">
+<style>
+.highlight path { stroke: orange; stroke-width: 2; }
+.highlight polygon:first-of-type { fill: orange; }
+.highlight polygon { stroke: orange; }
+.highlight text { fill: white; }
+</style>
+<script>
+<![CDATA[
+function clickHandler(event) {
+    const g = event.target.closest("g");
```

Comment:
Suggestion: You can automatically unhighlight previously highlighted elements by

```javascript
for (const elem of Array.from(document.getElementsByClassName('highlight'))) {
    elem.classList.remove('highlight'); }
```

Reply:
I often find myself tracing axes through the tree, highlighting the entire path, so I actually find it useful not to do this.